src/pages/admin/reports/components/funds-expenditures-section/FundsExpendituresSection.module.scss

```
@@ -26,13 +26,40 @@
     gap: 20px;
 }
 
+.disclaimer-header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
 .disclaimer-label {
     @include type.subtitle-1-medium;
     font-weight: 700;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)